firmware/thruna/DIM/CMakeLists.txt

```diff
@@ -15,7 +15,9 @@ file(GLOB_RECURSE HW_SRCS
     "${CMAKE_CURRENT_SOURCE_DIR}/src/hw/*.c"
     "${CMAKE_CURRENT_SOURCE_DIR}/src/cubemx/Src/*.c"
 )
-list(APPEND IO_SRCS "${SHARED_HW_SRCS}")
+list(APPEND HW_SRCS "${SHARED_HW_SRCS}")
+list(REMOVE_ITEM HW_SRCS "${SHARED_HW_INCLUDE_DIR}/hw_sdio_sd.c")
+
```

Comment:
I would prefer to see that the SD card drivers are opt-in rather than opt-out @gtaharaedmonds

Reply:
I think yea eventually we're going to have to change IO and HW level stuff to be all opt-in when we have more boards using shared IO code (i.e., when Quadruna boards get set up). However, I'd rather we do that switch properly all at once rather than in this PR. Since everything is opt-out at present, I'm fine leaving as is for now. How does that sound @Lucien950 @Jiajun-Huang ?